website/src/css/_colors.scss

```diff
@@ -0,0 +1,93 @@
+// Colors
+
+@mixin detox-colors-light($prefix) {
+  --#{$prefix}-r0: #f14c52ff;
```

Comment:
Remember what I told you earlier about generating colors from a SASS map variable.